kedro-telemetry/kedro_telemetry/plugin.py

```diff
@@ -159,8 +159,16 @@ def _get_project_properties(hashed_username: str, project_path: str) -> Dict:
         with open(pyproject_path) as file:
             pyproject_data = toml.load(file)
 
-        if "tools" in pyproject_data["tool"]["kedro"]:
-            properties["tools"] = pyproject_data["tool"]["kedro"]["tools"]
+        if "tool" in pyproject_data and "kedro" in pyproject_data["tool"]:
+            if "tools" in pyproject_data["tool"]["kedro"]:
+                # convert list of tools to comma-separated string
+                properties["tools"] = ", ".join(
+                    pyproject_data["tool"]["kedro"]["tools"]
+                )
+            if "example_pipeline" in pyproject_data["tool"]["kedro"]:
+                properties["example_pipeline"] = pyproject_data["tool"]["kedro"][
+                    "example_pipeline"
+                ]
```

Comment:
```suggestion
        if (build_config := pyproject_data.get("tool", {}).get("kedro", {}):
            if (tools := build_config.get("tools")):
                properties["tools"] = ", ".join(tools)
            if (example_pipeline := build_config.get("example_pipeline")):
                properties["example_pipeline"] = example_pipeline
```

? (didn't test the code, I believe it should work)

Reply:
Thank you, @astrojuanlu , it looks even better now! It seems like there was one parenthesis missing in the first line. I'll correct that and commit the changes.

Reply:
@astrojuanlu, I rolled back that change because I encountered an error in mocker related to that syntax and wasn't sure how to resolve the issue with the mocker. I suggest we leave it as it is for now. Error attached:

<img width="717" alt="Screenshot 2024-02-20 at 13 01 43" src="https://github.com/kedro-org/kedro-plugins/assets/40151847/bf99eeca-eebc-4304-a877-2eb36aa50633">